lib/Doctrine/DBAL/Driver/PDOPgSql/Driver.php

```diff
@@ -88,10 +88,12 @@ private function _constructPdoDsn(array $params)
 
         if (isset($params['dbname'])) {
             $dsn .= 'dbname=' . $params['dbname'] . ' ';
+        } elseif (isset($params['default_dbname'])) {
+            $dsn .= 'dbname=' . $params['default_dbname'] . ' ';
         } else {
```

Comment:
I rather want to remove the "else", that way we get the default behaviour (username) but people can still override it something like template1 if they want to live on the edge.